.editorconfig

```
@@ -0,0 +1,5 @@
+[*.{jelly, js, less, css, hbs}]
```

Since this is in the root of the repo. You ought to set `root = true`

https://editorconfig.org/#file-location
> When opening a file, EditorConfig plugins look for a file named .editorconfig in the directory of the opened file and in every parent directory. A search for .editorconfig files will stop if the root filepath is reached or an EditorConfig file with root=true is found.

```suggestion
root = true

[*.{jelly, js, less, css, hbs}]
```